pkg/jsonlog/jsonlog_test.go

```diff
@@ -1,62 +0,0 @@
-package jsonlog
```

Comment:
There was tests for `WriteLog`(written by me btw), now `WriteLog` removed. This package needs testing, but probably in other PR.